R/derive_var_anrind.R

```diff
@@ -29,47 +38,72 @@
 #' library(admiral.test)
 #' data(admiral_vs)
 #'
-#' ref_ranges <- tribble(
+#' ref_ranges1 <- tribble(
 #'   ~PARAMCD, ~ANRLO, ~ANRHI, ~A1LO, ~A1HI,
 #'   "DIABP",      60,     80,    40,    90,
 #'   "PULSE",      60,    100,    40,   110
 #' )
 #'
-#' admiral_vs %>%
+#' ref_ranges2 <- tribble(
+#'   ~PARAMCD, ~ANRLO, ~ANRHI,
+#'   "DIABP",      60,     80,
+#'   "PULSE",      60,    100
+#' )
+#'
+#' vs <- admiral_vs %>%
 #'   mutate(
 #'     PARAMCD = VSTESTCD,
 #'     AVAL = VSSTRESN
 #'   ) %>%
-#'   filter(PARAMCD %in% c("PULSE", "DIABP")) %>%
-#'   derive_vars_merged(ref_ranges, by_vars = exprs(PARAMCD)) %>%
-#'   derive_var_anrind() %>%
+#'   filter(PARAMCD %in% c("PULSE", "DIABP"))
+#'
+#' vs1 <- vs %>%
+#'   derive_vars_merged(ref_ranges1, by_vars = exprs(PARAMCD))
+#'
+#' vs2 <- vs %>%
+#'   derive_vars_merged(ref_ranges2, by_vars = exprs(PARAMCD))
+#'
+#' vs_anrind1 <- vs1 %>%
+#'   derive_var_anrind(use_a1hia1lo = TRUE) %>%
+#'   select(USUBJID, PARAMCD, AVAL, ANRLO:ANRIND)
```

Comment:
The displayed result does not contain any `"LOW LOW"` or `"HIGH HIGH"` values. I think we should create a small vs dataset using `tribble()` instead of using the admiral.test data.

Reply:
> The displayed result does not contain any `"LOW LOW"` or `"HIGH HIGH"` values. I think we should create a small vs dataset using `tribble()` instead of using the admiral.test data.

Agreed, have changed to use a `tribble()` vs dataset